wca-regulations.md

```diff
@@ -200,6 +200,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 7f1d) Surface: The flat surface on which the Stackmat has been placed. The mat is considered a part of the surface. The timer is not considered a part of the surface.
     - 7f2) The Stackmat timer must be attached to the mat and placed on the surface, with the timer on the side of the mat nearest to the competitor.
         - 7f2a) Exception: For Solving With Feet, the Stackmat must be placed directly on the floor. The timer device may be placed on the side of the mat farthest from the competitor.
+    - 7f3) The mat must have a line drawn parallel to the Stackmat timer, located 7 cm away from it, and measuring 10 cm.
```

Comment:
What does "parallel" for the curved Gen4 timer mean? Where do you measure the distance?